app/services/adapters/structs/income_details.rb

```diff
@@ -6,13 +6,17 @@ def employment_status
         employment_type || []
       end
 
+      def partner_employment_status
+        partner_employment_type || []
+      end
+
       def serializable_hash(options = {})
         super(
           options.merge(
-            methods: [:employment_status],
+            methods: [:employment_status, :partner_employment_status],
             # `employment_type` is the name for employment_status
             # in the datastore, we don't use it
-            except: [:employment_type, :dependants, :income_payments, :income_benefits]
+            except: [:employment_type, :partner_employment_type, :dependants, :income_payments, :income_benefits]
```

Comment:
Not sure if i should just be naming this `partner_employment_status` in the schema?

Reply:
The type was used in the datastore to determine which means assessment is required, rather than as a record of all employment relations. Personally, I think we should change the datastore to store all values as an array, similar to how it is done on Apply. We can then compute the work stream and value for the MAAT API within the datastore.

However, I don't think this change needs to be included in your current pull request.

Reply:
I'd be keen to consolidate employment_type and employment_status if we don't think we need 2 different attributes. It will simplify this point a lot